flash/image/classification/transforms.py

```diff
@@ -91,3 +93,30 @@ def train_default_transforms(image_size: Tuple[int, int]) -> Dict[str, Callable]
     else:
         transforms = {"per_sample_transform": ApplyToKeys(DataKeys.INPUT, T.RandomHorizontalFlip())}
     return merge_transforms(default_transforms(image_size), transforms)
+
+
+@dataclass
+class ImageClassificationInputTransform(InputTransform):
```

Comment:
Since we have this one, we could delete the other transforms in this file

Reply:
Yes, but there is a lot of dependencies in the documentation. Let's do it within another PR.